modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/js/CategorySelector.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+// @ts-ignore
```

Comment:
@p2kmgcl, why is this `ts-ignore`? Is there for muting an import line? Or is this a leftover?

Reply:
It's because AssetVocabularyCategoriesSelector is not written in typescript (yet) and that bugs the compiler. All `ts-ignores` are there because of that, otherwise I would need to migrate everything to ts :sweat_smile:

Reply:
Have you already tried to add [`allowJS`](https://www.typescriptlang.org/tsconfig#allowJs) in the `tsconfig` file? This should allow the import of js files in `ts` or `tsx` files.

Reply:
:thinking: you are right, that sounds like a better idea until we have migrated all dependencies. The tsconfig is autogenerated, let me check if it allows me to include that property :smile:

Reply:
I've been trying to add "allowJs" to tsconfig (both in 'asset-taglib' and 'commerce-product-content-web') and the compiler struggles finding the dependencies:

```
BUILD JS: No previous build detected.
src/main/resources/META-INF/resources/js/CategorySelector.tsx(15,49): error TS2307: Cannot find module 'asset-taglib' or its corresponding type declarations.
src/main/resources/META-INF/resources/js/TagSelector.tsx(15,33): error TS2307: Cannot find module 'asset-taglib' or its corresponding type declarations.
Command /home/p2kmgcl/Projects/community-portal/liferay-portal/modules/node_modules/@liferay/npm-scripts/node_modules/.bin/tsc --emitDeclarationOnly exited with code 2
```

I think this won't work :sweat:

Reply:
@rosdb do you want to give it a try or do we keep the ts-ignore?

Reply:
@p2kmgcl, for now, let's keep the `ts-ignore`, then we'll fix it at another time 👍🏻

Reply:
Alright, let's move on then. I am sorry but I spent like an hour messing with the tsconfig files and I don't know how to improve that :sweat: